Secondlesson/OperationsOnImages/ImageInfo.h

```diff
@@ -0,0 +1,24 @@
+//
+// Created by s1000633 on 5/17/21.
+//
+
+#ifndef OPENCV_TUTORIALS_IMAGEINFO_H
+#define OPENCV_TUTORIALS_IMAGEINFO_H
+
+#include <opencv2/opencv.hpp>
+#include <iostream>
+#include <string>
+
+class ImageInfo {
+    private:
+        int cols;
```

Comment:
I'd like to suggest you to use some style for naming of private members. You could use prefix 'm_' in front of name variable or for instance, you can use '_' as a postfix to name variable. This will help distinguish class member's variable from local variable